mysql-test/mroonga/storage/collation/utf8mb4_uca1400_ai_ci/r/french.result

```diff
@@ -0,0 +1,12 @@
+DROP TABLE IF EXISTS diaries;
+SET NAMES utf8mb4;
+CREATE TABLE diaries (
+content varchar(256) COLLATE utf8mb4_uca1400_ai_ci,
+FULLTEXT INDEX (content)
+) DEFAULT CHARSET=utf8mb4;
+INSERT INTO diaries VALUES ("Je suis un garçon.");
+INSERT INTO diaries VALUES ("L'homme n'est qu'un roseau");
```

Comment:
```suggestion
INSERT INTO diaries VALUES ("L'homme n'est qu'un roseau.");
```

BTW, in general, we should use single quote not double quote for string literal in SQL. Because SQL standard does so. (MySQL accepts both of single quote and double quote.)

Reply:
fix: [f72c39e](https://github.com/mroonga/mroonga/pull/908/commits/f72c39e818616641189dbe13026d020bc7defaf9), [8fb3a8d](https://github.com/mroonga/mroonga/pull/908/commits/8fb3a8db41d4c1e578173cfd6ec8fc71b3f2c231) I couldn't run the test after fixing the test case mm. Also, I fixed tests using the single quote